src/Tribe/Template.php

```diff
@@ -73,6 +73,15 @@ class Tribe__Template {
 	 */
 	protected $template_folder_lookup = false;
 
+	/**
+	 * Create a class variable for the include path, to avoid conflicting with extract.
+	 *
+	 * @since  4.11.0
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: d96f4570943b8464c84847879078cc6b</sub>